src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,70 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendarGap;
+
+  padding: $calendarPadding;
+  width: $daySquareSize * 7 + $calendarGap * 6;
+
+  font-size: 30px;
+  font-family: Arial , sans-serif;
+
+  @for $day from 1 through 31 {
+    &__day-#{$day} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      box-sizing: border-box;
+      width: $daySquareSize;
+      height: $daySquareSize;
+
+      background: #eee;
+      border: 1px solid #000;
+      cursor: pointer;
+      transition-duration: 350ms , 700ms ;
+      transition-property: transform, background-color ;
```

Comment:
since you are using transition duration same value more than once you could create a variable and store it in the variable.scss and replace it here.

Reply:
@Grayfox147 agree 👍